docs/tutorials/slack-history.md

```diff
@@ -0,0 +1,90 @@
+# Saving Slack Messages
```

Comment:
We will likely want to think about the title for SEO. I am sure @johnlafleur will have some ideas here!

Reply:
hahaha. i agree. open to whatever is most SEO-able.